core/controllers/question_editor.py

```diff
@@ -30,21 +30,20 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_ids):
+    def post(self, comma_separated_skill_ids):
         """Handles POST requests."""
-        skill_ids_list = skill_ids.split(',')
-        if len(skill_ids_list) > 3:
+        skill_ids = comma_separated_skill_ids.split(',')
+        if len(skill_ids) > constants.MAX_SKILLS_PER_QUESTION:
             raise Exception(
                 'More than three QuestionSkillLink for one '
```

Comment:
Update the exception string to reference constants.MAX_SKILLS_PER_QUESTION, in case that value changes in the future. (Just interpolate the number in the error message.)

Reply:
Done.